tests/ccgbugs/tcodegen_template_with_generic.nim

```diff
@@ -0,0 +1,30 @@
+discard """
+labels: "codegen error_compilation generics template"
+description: '''
+  . From https://github.com/nim-lang/Nim/issues/6756
+    Error in template when using the type of the parameter inside it
+  . The problem is in the line var res: A[T]. The assert fails because symbol is
+    A_105025: ["sfUsed"], [], ["issue.nim", 5, 15], skType
```

Comment:
```suggestion
  . The problem is in the line `var res: A[T]`. The assert fails because symbol is
    `A_105025: ["sfUsed"], [], ["issue.nim", 5, 15], skType`
```